frontend/packages/topology/src/actions/modify-application.ts

```diff
@@ -1,9 +1,9 @@
+import { RESOURCE_NAME_TRUNCATE_LENGTH } from '@console/shared/src/constants';
 import { KebabOption } from '@console/internal/components/utils';
 import { truncateMiddle } from '@console/internal/components/utils/truncate-middle';
 import { K8sResourceKind, K8sKind } from '@console/internal/module/k8s';
 import { ServiceModel as KnativeServiceModel } from '@console/knative-plugin';
-import { RESOURCE_NAME_TRUNCATE_LENGTH } from '../const';
-import { editApplicationModal } from '../components/modals';
+import { editApplicationModal } from '@console/topology/src/components/modals';
```

Comment:
Nit:
```suggestion
import { editApplicationModal } from '@console/topology';
```
& sort please